dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1749,36 +1744,30 @@ def _get_quantiles(self) -> None:
         ]
         self.quantiles = self._get_percentile(percentiles=percentiles)
 
-    def _update_helper(self, df_series_clean: pd.Series, profile: dict) -> None:
+    def _update_helper(self, df_series_clean: pl.Series, profile: dict) -> None:
         """
         Update base numerical profile properties w/ clean dataset and known null params.
 
         :param df_series_clean: df series with nulls removed
-        :type df_series_clean: pandas.core.series.Series
+        :type df_series_clean: polars.series.series.Series
         :param profile: numerical profile dictionary
         :type profile: dict
         :return: None
         """
-        self._greater_than_64_bit = (
-            not df_series_clean.empty
-            and df_series_clean.apply(pd.to_numeric, errors="coerce").dtype == "O"
-        )
+        self._greater_than_64_bit = df_series_clean.dtype == pl.Object
         if self._greater_than_64_bit:
-            df_series_clean = df_series_clean.to_numpy()
-            df_series_clean = df_series_clean[df_series_clean != np.nan]
-            if df_series_clean.size == 0:
+            df_np_series_clean = df_series_clean.to_numpy()
+            df_np_series_clean = df_np_series_clean[df_np_series_clean != np.nan]
```

Comment:
is this saying that there is always a 0 or 1 for index values?

Reply:
may not be a question for this PR

Reply:
If i am reading it correctly it is just dropping any nans in the array